### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by banker**
A few items:

1. Please see JsonSearchTest.java. It seems that client.jsonSet() does not work as I would expect with JSONObject objects (it seems to serialize these objects as maps, adding the key "map" to stored object). I assume this is a minor bug, but the example in the README reflects this usage. If we don't fix this bug, we'll need to change the README to reflect this.

2. Please change the "About" the project in GitHub to "Jedis is a Java client for Redis designed for performance and ease of use." 

3. Do we still need to include Yourkit as one of the sponsors? Are they still sponsoring?

### CODE REVIEW

**Comment by chayim** [comment]
Thanks @banker I updated #2. 

@sazzad16 Please take care of #1.

**Comment by sazzad16** [comment]
@banker 

1. RedisJSON commands here are just reimplementation of JRedisJSON.
  JRedisJSON was based on Gson for only POJOs. So do currently implemented `json...` methods.
  JSONObject could be a part of 4.1 but it is not planned to be a part of 4.0

**Comment by banker** [comment]
@chayim Thanks for taking care of #2, but can you please add a "." at the end?

**Comment by banker** [comment]
@sazzad16 Thanks for letting me know. I've update the example to use GSON.

File: README.md [hunk 1]
```diff
@@ -7,142 +9,94 @@
...
 [![Discord](https://img.shields.io/discord/697882427875393627?style=flat-square)](https://discord.gg/qRhBuY8Z)
 
-# Jedis
-
-Jedis is a blazingly small and sane [Redis](http://github.com/antirez/redis "Redis") java client.
-
-Jedis was conceived to be EASY to use.
-
-Jedis is fully compatible with redis 2.8.x, 3.x.x and above*.
-
```
Comment (yangbodong22011) [inline README.md:17]: `blazingly small and sane` is a brilliant description and doesn’t need to be modified.

File: README.md [hunk 2]
```diff
@@ -7,142 +9,94 @@
...
-    </repository>
-  </repositories>
+```java
+JedisPool pool = new JedisPool("localhost", 6379);
 ```
 
-and
+Once you have a `JedisPool` instance, you can use a [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) block to get a connection and run Redis commands.
 
-```xml
```
Comment (yangbodong22011) [inline README.md:78]: Why cancel the version number prompt, this is user-friendly.

File: README.md [hunk 3]
```diff
@@ -7,142 +9,94 @@
...
-  </dependencies>
+## Using RedisJSON and Redis modules
+
+Jedis provides support for some of the [Redis modules](https://redis.io/modules), most notably [RedisJSON](https://oss.redis.com/redisjson/) and [RediSearch](https://oss.redis.com/redisearch/).
+
+The latest versions of RedisJSON let you store, manipulate, index, and query JSON.
+ To 
+use these features with Jedis, you'll need to use the `UnifiedJedis` interface. 
+See the [RedisJSON Jedis Quick Start](docs/redisjson.md) for details.
+
```
Comment (yangbodong22011) [inline README.md:54]: We only need a document link on the Redis Module page, and put all the specific information such as RedisJSON, RedisSearch in it, please stay sensible.

**Comment by chayim** [comment]
> @chayim Thanks for taking care of #2, but can you please add a "." at the end?

Done like dinner.

**Comment by banker** [comment]
@sazzad16 Can you look into this build failure? I don't think of because of any changes in this PR.

[ERROR] Tests run: 20, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.568 s <<< FAILURE! - in redis.clients.jedis.tests.commands.ClientCommandsTest
[ERROR] killSkipmeYesNo2(redis.clients.jedis.tests.commands.ClientCommandsTest)  Time elapsed: 0.017 s  <<< FAILURE!

**Comment by sazzad16** [comment]
@banker Sorry, we had to squash the branch. I have rebased your works here https://github.com/sazzad16/jedis/tree/banker-docs-update without _"Use GSON in example"_ as this has become optional after rework for RedisJSON 2.

**Pull request closed without merging**